code/modules/projectiles/guns/energy.dm

```diff
@@ -490,17 +490,18 @@
 /datum/lasrifle/base/standard
 	charge_cost = 20
 	ammo = /datum/ammo/energy/lasgun/M43
-	fire_delay = 3
+	fire_delay = 2
 	fire_sound = 'sound/weapons/guns/fire/laser3.ogg'
 	message_to_user = "You set the Lasrifle's charge mode to standard fire."
-	icon_state = "tx73"
+	fire_mode = GUN_FIREMODE_AUTOMATIC
+	icon_state = "tx73_auto"
```

Comment:
The two modes being both the same sprite is bad. either make a new one, or make a choice as to which one should be which.

Reply:
done